resources/base-runtime-mock.cfg

```diff
@@ -28,7 +28,7 @@ mdpolicy=group:primary
 
 [demorepo]
```

Comment:
I didn't think of that; not a bad idea. Although it's all still very unofficial :) I'll leave that to you.